### ISSUE TITLE
[Feature Request] API for IsWindowMinimised

### ISSUE THREAD

**Issue by Elypha** [NightmareXIV/NotificationMaster#21]
Hello,

Thanks for your great work as always!

I'm thinking about an API that returns a bool to reflect if the current game window is minimised or not. It seems that it's not a part of the current API if I didn't miss anything x

Do you think it's possible?

**Comment by Limiana** [comment]
Hello!
You can get it like this:
```
WINDOWINFO info = new();
if(TryFindGameWindow(out var hwnd) && User32.GetWindowInfo(hwnd, ref info))
{
    return ((WindowShowStyle)info.dwStyle).HasFlag(WindowShowStyle.SW_MINIMIZE);
}
```
TryFindGameWindows is in ECommons, you can copy it, and WINDOWINFO is in PInvoke.User32.dll which is shipped with Dalamud

**Comment by Elypha** [comment]
@Limiana Thanks a lot for the very detailed reply!
I tried `SW_MINIMIZE` but in either case (windowedvs. minimised) the flag was not set. the documentation says 

WS_MINIMIZE | 0x20000000L | The window is initially minimized. Same as the WS_ICONIC style.
-- | -- | --

so I wonder if they literally mean it by 'initially'.

I tried this one from NotificationMaster and it worked:

```C#
    [DllImport("user32.dll")]
    [return: MarshalAs(UnmanagedType.Bool)]
    private static extern bool IsIconic(IntPtr hWnd);
```

yet another time having trouble with win32 and appreciate the information very much!
